### PR TITLE
[SMALLFIX]Update MetricsConfigTest.java

### PR DESCRIPTION
**Pull request by NJUaaron**
Changed some sentences corresponding to the package "org.junit.Assert".

### CODE REVIEW

**Comment by newnius** [comment]
Jenkins, test this please

File: core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java [hunk 1]
```diff
@@ -11,7 +11,8 @@
 
 package alluxio.metrics;
 
-import org.junit.Assert;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
 import org.junit.Before;
```
Comment (newnius) [inline core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java]: > /usr/src/alluxio/core/common/src/test/java/alluxio/metrics/MetricsConfigTest.java:16: error: 'org.junit.Before' should be separated from previous import group.

**Comment by aaudiber** [comment]
ping @NJUaaron

**Comment by aaudiber** [comment]
alluxio-bot, merge this please

**Pull request merged**